### PR TITLE
format `react` folder with prettier

### PR DESCRIPTION
**Pull request by phryneas**
Well, this is it. It's a lot to look at, but mostly I didn't see meaningful "change of intent" through the formatting.

I'll highlight some places.

PS: the `useSuspenseQuery` tests were already failing before my changes. Not sure what's going on there.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -166,7 +166,7 @@
     "bracketSpacing": true,
     "printWidth": 80,
     "semi": true,
-    "singleQuote": true,
+    "singleQuote": false,
```
Comment (phryneas) [inline package.json:169]: We had this at `true` before, but we agreed on `"` instead of `'` as defaults, so here we go :)

**Comment by jerelmiller** [comment]
Would it be possible to move that [separate commit](https://github.com/apollographql/apollo-client/pull/10661/commits/9e6f86d677512a9cf369a59607fcf441fbda9e92) to its own PR? While I respect the change, I'd find this a bit easier to review if it was _just_ formatting and I didn't have to think through the changes to the `waitFor`s.

Also, do you know why this PR has commits from renovate? I found that a bit odd. Perhaps we need to merge `main` into `release-3.8`?

**Comment by phryneas** [comment]
@jerelmiller the renovate-commits are from `main` - I did merge `main` into `3.8` locally to be "the most up-to-date" possible.

As for the separate commit: I found that without that, the files get a *lot* longer - the affected files are all multiple hundred lines longer as prettier went from 3 lines to 5 lines with each `waitFor`. So I pretty much started that work to make things easier to review.  
But if you still insist, I can of course move that out - just tomorrow. It's 7:30pm here ;)

**Comment by jerelmiller** [comment]
@phryneas thanks for the context around that!

I think for me its less about lines of code and more around _intent_. I'd love to review this PR with the _intent_ that its all related to formatting. I'd love to be able to see the changes to `waitFor` (and others) in isolation of the formatting changes so its easier to wrap my head around how they affect the tests.

Totally fine if you'd like to do it tomorrow 😄.

**Comment by alessbell** [comment]
> [...] the renovate-commits are from `main`

Just updated the `release-3.8` branch against `main` 👍

**Comment by phryneas** [comment]
@jerelmiller I've moved that one commit out into https://github.com/apollographql/apollo-client/pull/10662

**Comment by jerelmiller** [comment]
@phryneas I realize its been a bit since this PR was opened and apologies for waiting so long to review. 

Looks like there are quite a few conflicts. Would it be worth re-opening this PR with a new branch off 3.8? If not, I'd be happy to re-review once conflicts are resolved.

**Comment by jerelmiller** [comment]
Its been some time and this PR is incredibly out-of-date at this point. I think it would be easier to create a new PR instead of trying to fix all the merge conflicts so I'm going to close this.

**Pull request closed without merging**